docs/tutorial.md

```diff
@@ -91,31 +90,30 @@ NAME
 echo
 ```
 
-The status of type `Succeeded` shows that the `Task` ran successfully.
+The `Succeeded` status confirms that the `Task` completed with no errors.
+
+To see more detail about the execution of your `Task` view its logs:
 
-To see the actual outcome, use the following command:
 ```bash
 tkn taskrun logs echo-hello-world-task-run
 ```
 
-You will get an output similar to this:
+Below is the example output:
```

Comment:
```suggestion
You will get output similar to the following:
```

Reply:
Same as above. This is less wordy than your suggestion.

Reply:
Done.